### PR TITLE
tweak user warning message to help users when making a new install

### PR DESCRIPTION
**Pull request by Charlotte-Morgan**
### What does it fix?

* Ticket: #
* Funded by: 
* Description: 
  * 

### Checklist:

- [ ] Commit messages are descriptive and explain the rationale for changes
- [ ] Add to the changelog in metadata.txt if it's a new feature
- [ ] Unit test for new code added
- [ ] Request someone to review or test your PR

### CODE REVIEW

**Comment by Gustry** [comment]
Hi @Charlotte-Morgan, nice to see you again.

Looks good to me about your new text. But in the PR, there is also safe_extra/parameters which is updated. I don't think this is expected.

**Comment by Charlotte-Morgan** [comment]
@Gustry - thank you. Let me try again.

**Comment by Gustry** [comment]
I can do it if you need ;-)
